src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -150,6 +163,35 @@ public static void main(String[] args) {
     System.exit(0);
   }
 
+  List<String> parseDepgraphRoots() {
+    List<String> result = new ArrayList<>();
+    for (String file : opts.depgraphFiles) {
+      try (FileReader reader = new FileReader(new File(file))) {
+        List<List> list = new Gson().fromJson(reader, new TypeToken<List<List>>(){}.getType());
```

Comment:
GSON is pretty neat at mapping into a Java class that represents the structure and fields. Ultimately we'll probably need that, would it make sense to just do that now? It'd also make this code more readable IMHO.